lib/spack/spack/test/cmd/versions.py

```diff
@@ -24,50 +23,50 @@ def test_safe_versions():
     versions('--safe', 'zlib')
 
 
-@pytest.mark.network
+@pytest.mark.maybeslow
 def test_remote_versions():
     """Test a package for which remote versions should be available."""
 
     versions('zlib')
 
 
-@pytest.mark.network
+@pytest.mark.maybeslow
 def test_remote_versions_only():
     """Test a package for which remote versions should be available."""
 
     versions('--remote', 'zlib')
 
 
-@pytest.mark.network
+@pytest.mark.maybeslow
 @pytest.mark.usefixtures('mock_packages')
 def test_new_versions_only():
     """Test a package for which new versions should be available."""
 
     versions('--new', 'brillig')
 
 
-@pytest.mark.network
+@pytest.mark.maybeslow
 def test_no_versions():
     """Test a package for which no remote versions are available."""
 
     versions('converge')
 
 
-@pytest.mark.network
+@pytest.mark.maybeslow
 def test_no_unchecksummed_versions():
     """Test a package for which no unchecksummed versions are available."""
 
     versions('bzip2')
 
 
-@pytest.mark.network
+@pytest.mark.maybeslow
```

Comment:
Would it be useful to leave the `.network` mark on as well? It seems like useful metadata.